src/components/Confetti/Confetti.js

```diff
@@ -0,0 +1,12 @@
+const props = {
+  confettiParticles: {
+    type: Number,
```

Comment:
Please update this to allow both `Number` and `String` (otherwise consumers would always have to use `:particles="300"`  instead of `particles="300"`. You'll need to add some logic in that case to parse from String when needed (let me know if you'd like any help)

```js
type: [Number, String],
```